src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -156,10 +158,17 @@ consistent "has a" relationship is tracked by this ticket: https://broadworkbenc
     email: WorkbenchEmail,
     roles: Set[ResourceRoleName],
     actions: Set[ResourceAction],
+    descendantPermissions: Set[AccessPolicyDescendantPermissions],
     public: Boolean)
     extends WorkbenchGroup
 
-@Lenses final case class AccessPolicyMembership(memberEmails: Set[WorkbenchEmail], actions: Set[ResourceAction], roles: Set[ResourceRoleName])
+@Lenses final case class AccessPolicyDescendantPermissions(resourceType: ResourceTypeName, actions: Set[ResourceAction], roles: Set[ResourceRoleName])
+@Lenses final case class AccessPolicyMembership(memberEmails: Set[WorkbenchEmail],
+                                                actions: Set[ResourceAction],
+                                                roles: Set[ResourceRoleName],
+                                                descendantPermissions: Option[Set[AccessPolicyDescendantPermissions]] = Option(Set.empty)) {
```

Comment:
I changed the default value from `None` to `Option(Set.empty)` so that I wouldn't have to change the v1 routes specs at all. With `None` as the default value, the expected response would be `AccessPolicyMembership(_, _, _, None)` but the response returned by the ResourceService is `AccessPolicyMembership(_, _, _, Some(Set.empty))`